ui/app/src/main/java/com/example/fuel/api/AccountApiService.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.fuel.api
+
+import com.example.fuel.model.User
+import retrofit2.Response
+import retrofit2.http.*
+
+interface AccountApiService {
+
+    @FormUrlEncoded

```

Comment:
This can't work. Endpoint is expecting that you will send payload in body not in url. You need to change this to something like this:

![image](https://user-images.githubusercontent.com/61120635/201299825-2ff5cef3-fe12-446e-9d2c-66fe727135f3.png)